backend/apps/slack/admin/entity_channel.py

```
@@ -0,0 +1,14 @@
+from django.contrib import admin
+from django.contrib.contenttypes.admin import GenericTabularInline
```

Ruff (0.12.2)

`django.contrib.contenttypes.admin.GenericTabularInline` imported but unused

Remove unused import: `django.contrib.contenttypes.admin.GenericTabularInline`

(F401)